packages/cli/src/cli/config/action.import.ts

```diff
@@ -128,4 +147,74 @@ export class CommandImport extends CommandLineAction {
 
     this.promises.push(promise);
   }
+
+  async outputChange(output: string, mem: BasemapsConfigProvider, cfg: BasemapsConfigProvider): Promise<void> {
+    const inserts: string[] = ['# New Layers\n'];
+    const updates: string[] = ['# Updates\n'];
+    const id = 'ts_aerial';
+    const newData = await mem.TileSet.get(id);
+    const oldData = await cfg.TileSet.get(id);
+    if (newData == null || oldData == null) throw new Error('Failed to fetch aerial config data.');
+    for (const layer of newData.layers) {
+      const existing = oldData.layers.find((l) => l.name === layer.name);
+      if (layer.name === 'chatham-islands_digital-globe_2014-2019_0-5m') continue; // Ignore duplicated layer.
```

Comment:
should do this before the `.find()` as your not using the `existing` value here.